### ISSUE TITLE
remove ConstDefaultRows constant

### ISSUE THREAD

**Issue by jokin** [SolrNet/SolrNet#377]
Currently, there is a ConstDefaultRows = 100000000 which sets a default high number of rows to mimic sql, and could lead to a big performance hit. I Think that the library should respect the solr server default, which now  It's 10, but It can be changed in the solr configuration if someone need to. 

I can make the patch and it can be integrated in a future 1.1 release for example to advise of a breaking change.

**Comment by gjunge** [comment]
@jokin I agree with you, and I strongly doubt if anybody ever uses the SolrNet default. With other words I wonder if it is really a breaking change and if we need to wait to version 1.1 with it. 
I remember running into this issue myself when I just started out with the library so I’m strongly opinionated in favor of changing this sooner than later. 
@xavier-morera ?

**Comment by xavier-morera** [comment]
Agreed with both, 10 makes sense. Setting rows, just like wt, is one of those things that you should have a default that makes sense (wt already fixed)

@jokin this is an easy one and I see you have contributed since 2013. Would you like to provide a PR?

@gjunge sooner is better as you mention.

**Comment by jokin** [comment]
Yes, knowing that there's interest in removing the default I will provide a PR with the change and tests updated. 

Anyway, I will try to remove the default and don't send the parameter if the rows number is
not set, so the solr server default prevails, because it can be 20 or 50, and in that case I don't think that solrnet should send the parameter at all.
